ocs_ci/ocs/registry.py

```diff
@@ -279,3 +272,23 @@ def image_rm(registry_path):
     ocp_obj = ocp.OCP()
     ocp_obj.exec_oc_debug_cmd(node=master_list[0], cmd_list=cmd_list)
     logger.info(f"Image {registry_path} rm successful")
+
+
+def check_image_in_registry(image_url):
+    """
+    Function to check either image present in registry or not
+
+    Args:
+        image_url (str): Image url to be verified
+
+    Returns:
+        True : Returns True if present
+
+    """
+    output = image_list_all()
+    output = output.split("\n")
+    if any(image_url in i for i in output):
+        logger.info("Image URL present")
+        return True
+    else:
```

Comment:
else  is not necessary when return statement is present inside if condition.
raise UnexpectedBehaviour can be written outside else.